Examples/font-selector.lisp

```diff
@@ -107,7 +107,7 @@
 
 (defun face-changed (pane value)
   (declare (ignore pane))
-  (let ((sizes (if value (font-face-all-sizes value) nil)))
+  (let ((sizes (and value (font-face-all-sizes value))))
```

Comment:
this is also semantically incorrect:
`(if value sizes nil)` means if there is value, return all sizes, otherwise nil
`(and value sizes)` means return whenever both value and sizes are non-NIL
while it doesn't make a difference to the compiler in case of Common Lisp, it does for a reader

Reply:
> this is also semantically incorrect:

Given that the forms are semantically equivalent, it is incorrect to say it is semantically incorrect. The truth table for the value sizes is bound to are identical in both cases. `AND` is an idiomatic way to write one line `when`s. I agree it does make a different to the reader, it maintains a consistent way to separate clauses inside of `if`s and `when`s, using new lines.